zng/docs/spec.md

```diff
@@ -351,27 +351,26 @@ previously defined type, the appropriate typedef control code must
 be re-emitted
 (and note that the typedef may now be assigned a different ID).
 
-### 3.1.3 Compressed Message Block
+### 3.1.3 Compressed Value Message Block
 
-Following a header byte of 0x86 is a compressed message block.  Such a
-block comprises a compressed sequence of control and value messages.
-The sequence can include any message except for end-of-stream markers
-and compressed message blocks.
+Following a header byte of 0x86 is a compressed value message block.
+Such a block comprises a compressed sequence of value messages.  The
+sequence must not include control messages.
```

Comment:
I made this ZNG specification change in anticipation of adding Boyer-Moore string search inside zio/zngio.zngScanner.scanUncompressed. Excluding control messages like this means we can immediately discard a block if the search fails, without looking for control messages.

It accounts for the the changes in ZNG outputs and the zng-output-check failure.